packages/firebase/index.d.ts

```diff
@@ -15,6 +15,8 @@
  * limitations under the License.
  */
 
+import { EmulatorMockTokenOptions } from '@firebase/util';
```

Comment:
It is supposed to be a self-contained typing file, so should not reference other packages. Can you copy the definition here, so people can access the type using `firebase.firestore.EmulatorMockTokenOptions` ?

Reply:
Done!